exercises/numbers/remainderOf/remainderOf.js

```diff
@@ -15,14 +15,18 @@
  * @param {number} num The input number
  * @returns {boolean} True if num is even and false otherwise
  */
-function remainderOf(num) {
+function remainderOf(num1,num2){
```

Comment:
Nice solution! `return num1 % num2` would also solve it. `%` is called the modulo operator, and gives you the remainder of `num1/num2` directly.